### ISSUE TITLE
Trailing whitespace messes up parse_wp_config

### ISSUE THREAD

**Issue by bitwombat** [wp-cli/wp-config-transformer#7]
If I have this as my wp-config.php
```
<?php
define('WP_CACHE', true); //
define('DB_NAME', 'wtngcoma_wtng');
```

This is what $config ends up being in WPConfigTransformer->parse_wp_config:
```
<?php\ndefine('WP_CACHE', true); define('DB_NAME', 'wtngcoma_wtng');
```
See that space before the second define?  That messes up the preg_replace in `update`.

I can write a test and attempt to fix it, but I'm not sure where to put the test.  Some guidance there and/or opinion about this bug?

**Comment by fjarrett** [comment]
@bitwombat Thanks for sharing this use case. I believe this falls under a known issue. The current implementation can only match one definition per line.

https://github.com/wp-cli/wp-config-transformer/blob/master/README.md#known-issues

**Comment by bitwombat** [comment]
But there is only one definition per line...

I think you're confusing my original wp-config.php input, which has one definition per line, with the $config string that parse_wp_config makes.  See above.

**Comment by fjarrett** [comment]
@bitwombat ah that makes sense. What version of WP-CLI are you using? I just tried to reproduce locally with 2.0.1 and couldn't.

**Comment by bitwombat** [comment]
I'm using 2.0.1

Try with my original text up there verbatim.  The trailing // is what causes the space.

Then:

```
$ wp config set DB_NAME fjarrett
Success: Updated the constant 'DB_NAME' in the 'wp-config.php' file with the value 'fjarrett'.
$ cat wp-config.php
<?php
define('WP_CACHE', true); //
define('DB_NAME', 'wtngcoma_wtng');
```

**Comment by bitwombat** [comment]
Ah bloody @#(!  @fjarrett's commit on master (#ad435a) fixed it back in April!  Just not released yet.

Though the change wasn't intended to fix this problem, so I'll probably still write a test.

**Comment by bitwombat** [comment]
#8

**Comment by danielbachhuber** [comment]
> Ah bloody @#(! @fjarrett's commit on master (#ad435a) fixed it back in April! Just not released yet.

Sorry about that, @bitwombat :( I'll tag a 1.2.2 release when your test passes.

**Comment by bitwombat** [comment]
My test passes now because of #ad435a.  So should be all set then #8 gets merged?

**Comment by schlessera** [comment]
Fixed in v1.2.2
